assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -3868,6 +3876,7 @@ protected String build_instructor_grade_submission_context(VelocityPortlet portl
         if (useSakaiGrader) {
             return template + TEMPLATE_INSTRUCTOR_GRADE_SUBMISSION_WITH_GRADER;
         } else {
+            context.put(RUBRICS_EXPORT_PDF, serverConfigurationService.getString(RUBRICS_EXPORT_PDF_PROPERTY));
```

Comment:
Not sure this should be enabled in assignments.

It also shouldn't be a string.

Reply:
Done.

Reply:
To show the pdf button, although the button is in a rubrics component, I have to get the server variable in Assignments.